config.json

```
@@ -589,6 +589,24 @@
                 "arrays",
                 "bitwise_operations"
             ]
-        }
+        },
+	{
+	    "uuid":"bf5c1f49-963a-4397-bb67-b58e1b7d45e3",
+	    "slug":"matching-brackets",
+	    "core":true,
```

```suggestion
	    "core":false,
```

For now, we don't want to add any core exercises to not put even more work on the mentors. I'll have to think about where this fits into the progression. I think we can leave it as floating exercise that the students can do whenever they want for now.